Havit.Blazor.Components.Web.Bootstrap/Grids/HxPaginationContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Havit.Blazor.Components.Web.Bootstrap;
+
+/// <summary>
+/// Pager.<br />
+/// Full documentation and demos: <see href="https://havit.blazor.eu/components/HxPaginationContext">https://havit.blazor.eu/components/HxPaginationContext</see>
+/// </summary>
+public class HxPaginationContext
```

Comment:
We exclusively use the `Hx` prefix for the components themselves; all supporting types do not use it. This should most likely be named `GridPaginationTemplateContext`.